streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -773,6 +774,10 @@ public void run() {
         try {
             runLoop();
             cleanRun = true;
+        } catch (final TimeoutException e) {
+
+            throw new TimeoutException("The thread-consumer : " + threadMetadata.consumerClientId() + " and restore-consumer : " + threadMetadata.restoreConsumerClientId() + "has happened TimeoutExceptio" +
```

Comment:
typo in the end: `has happened TimeoutExceptio`

Reply:
I modified the TimeoutException